<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>AddEditCoffeeForm</class>
    <widget class="QDialog" name="AddEditCoffeeForm">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>400</width>
                <height>300</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Добавить/Редактировать кофе</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
            <item>
                <widget class="QLineEdit" name="nameInput">
                    <property name="placeholderText">
                        <string>Название сорта</string>
                    </property>
                </widget>
            </item>
            <item>
                <widget class="QComboBox" name="roastInput"/>
            </item>
            <item>
                <widget class="QComboBox" name="typeInput"/>
            </item>
            <item>
                <widget class="QLineEdit" name="flavorInput">
                    <property name="placeholderText">
                        <string>Описание вкуса</string>
                    </property>
                </widget>
            </item>
            <item>
                <widget class="QSpinBox" name="priceInput">
                    <property name="minimum">
                        <number>0</number>
                    </property>
                    <property name="maximum">
                        <number>10000</number>
                    </property>
                </widget>
            </item>
            <item>
                <widget class="QSpinBox" name="volumeInput">
                    <property name="minimum">
                        <number>0</number>
                    </property>
                    <property name="maximum">
                        <number>1000</number>
                    </property>
                </widget>
            </item>
            <item>
                <widget class="QPushButton" name="saveButton">
                    <property name="text">
                        <string>Сохранить</string>
                    </property>
                </widget>
            </item>
        </layout>
    </widget>
</ui>